client/components/jetpack/wpcom-business-at/index.tsx

```diff
@@ -49,6 +52,7 @@ import 'blocks/eligibility-warnings/style.scss';
  */
 import JetpackBackupSVG from 'assets/images/illustrations/jetpack-backup.svg';
 import JetpackScanSVG from 'assets/images/illustrations/jetpack-scan.svg';
+import isJetpackSite from '../../../state/sites/selectors/is-jetpack-site';
```

Comment:
This should be under the internal dependencies, and should be just `state/site...`.

Reply:
yes! IDE autocompletion. Thanks

Reply:
This still needs to be fixed!